GraphEditor/GraphEditor.cpp

```diff
@@ -158,11 +158,12 @@ QString GraphEditor::newId(const QString &hint, const QStringList &blacklist) co
 
     //find a reasonable name and index
     size_t index = 0;
-    QRegExp rx("(.*)(\\d+)"); rx.indexIn(idBase);
-    if (rx.captureCount() == 2 and not rx.cap(1).isEmpty() and not rx.cap(2).isEmpty())
+    QRegularExpression rx("(.*)(\\d+)");
+    QRegularExpressionMatch rm = rx.match(idBase);
+    if (rx.captureCount() == 2 and not rm.captured(1).isEmpty() and not rm.captured(2).isEmpty())
```

Comment:
note that captureCount() is called against rx and not rm -- my understanding is it doesn't actually ensure that it *matched* 2 items, just that the regular expression itself contains two match groups. Could be removed.